### PR TITLE
Update the link to the openshift page

### PR DESCRIPTION
**Pull request by mscherer**
The old one was 404.

## Proposed Changes 

-  Make the link to openshift go to a more recent (4.7 => 4.10) and existing page

### CODE REVIEW

**Review by abrennan89 — APPROVED** [review]
/lgtm

thanks!

**Comment by abrennan89** [comment]
@mscherer I think you need to sign the CLA before this can merge

**Comment by psschwei** [comment]
@mscherer please let us know when you've signed the CLA so we can unhold this

File: docs/install/knative-offerings.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Here is a list of commercial Knative products (alphabetically):
...
 - [Nutanix Karbon](https://github.com/nutanix/karbon-platform-services/tree/master/automation/infrastructure/terraform/kcs/install_knative): Extend Nutanix Karbon with serverless capabilities by installing Knative on a Karbon managed Kubernetes cluster.
-- [Red Hat Openshift Serverless](https://docs.openshift.com/container-platform/4.7/serverless/serverless-getting-started.html): enables stateful, stateless, and serverless workloads to all run on a single multi-cloud container platform with automated operations. Developers can use a single platform for hosting their microservices, legacy, and serverless applications.
+- [Red Hat Openshift Serverless](https://docs.openshift.com/container-platform/4.10/serverless/discover/about-serverless.html): enables stateful, stateless, and serverless workloads to all run on a single multi-cloud container platform with automated operations. Developers can use a single platform for hosting their microservices, legacy, and serverless applications.
```
Comment (abrennan89) [inline docs/install/knative-offerings.md:17]:
```suggestion
- [Red Hat Openshift Serverless](https://docs.openshift.com/container-platform/4.11/serverless/discover/about-serverless.html): enables stateful, stateless, and serverless workloads to all run on a single multi-cloud container platform with automated operations. Developers can use a single platform for hosting their microservices, legacy, and serverless applications.
```

**Comment by abrennan89** [comment]
@mscherer reminder to sign the CLA please so that we can merge this

**Comment by abrennan89** [comment]
Updated to 4.11 which is actually the latest link

**Pull request closed without merging**